jara.gemspec

```diff
@@ -15,12 +15,19 @@ Gem::Specification.new do |s|
   s.summary       = %q{Builds and publishes JAR artifacts}
   s.description   = %q{Build self-contained JAR artifacts and publish them to S3}
 
-  s.files         = Dir['lib/**/*.rb', 'README.md', 'LICENSE.txt', '.yardopts']
+  s.files         = Dir['bin/*', 'lib/**/*.rb', 'README.md', 'LICENSE.txt', '.yardopts']
   s.require_paths = %w(lib)
+  s.bindir        = 'bin'
+  s.default_executable = s.name
 
-  s.platform = 'java'
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_dependency 'puck'
-  s.add_dependency 'aws-sdk-core'
+  if defined? JRUBY_VERSION
+    s.platform = 'java'
+    s.add_runtime_dependency 'puck'
+  else
+    s.platform = Gem::Platform::RUBY
+  end
```

Comment:
It's really hard to explain in the readme how Jara is primarily a tool for building standalone JRuby applications, but to do that you need to install another gem :dancers: